main.js

```diff
@@ -33,21 +32,31 @@ class Car {
       this.move()
     }, 16)
   }
+  stop() {
+    for (let i = 0; i < 100; i++) {
+      window.clearInterval(i)
+    }
+  }
 }
 
 const image = document.createElement('img')
 image.src = 'https://opengameart.org/sites/default/files/simple-travel-car-top_view.svg'
 image.setAttribute("width", "200px")
 image.setAttribute("height", "100px")
-image.setAttribute("position", "relative")
-
 document.body.appendChild(image)
 
 const ferarri = new Car(image, 5, 'east', [0, 0])
 
+let intervalState = false;
```

Comment:
Instead of having a global variable for this information, make your car responsible for knowing it. That way you can just ask the car if it's started or not.